### PR TITLE
Initial work on adding documentation for Radius metrics

### PR DESCRIPTION
**Pull request by ytimocin**
Thank you for helping make the Radius documentation better!

**Please follow this checklist before submitting:**

- [ ] [Read the contribution guide](https://docs.radapp.dev/contributing/docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

### CODE REVIEW

File: docs/content/operations/control-plane/observability/metrics/_index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (youngbupark) [inline docs/content/operations/control-plane/observability/metrics/_index.md:1]: @ytimocin You took the wrong dashboard screenshot. I put all new dashboards into `Project Radius` folder, not `General`

Use the following links. I set the time range to see all metrics. 

* Radius Overview - https://radius-longhaul-wus3-awejb5drbjabh8h0.wus3.grafana.azure.com/d/Wh5JpyxVz/radius-overview?orgId=1&from=1677877199000&to=1677887999000
* Resource Provider - https://radius-longhaul-wus3-awejb5drbjabh8h0.wus3.grafana.azure.com/d/Wh5JpyxVz/radius-overview?orgId=1&from=1677877199000&to=1677887999000

File: docs/content/operations/control-plane/observability/metrics/grafana.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+{{< tabs "Core RP & Link RP" "Deployment Engine" "UCP" >}}
+
+{{% codetab %}}
+The `grafana-system-services-dashboard.json` template shows Radius Core RP and Link RP statuses, including runtime, server-side, and async operation health:
+
+<img src="corerp-linkrp-grafana-dashboard.png" alt="Screenshot of the Core RP and Link RP dashboard" width=1200>
+{{% /codetab %}}
+
+{{% codetab %}}
+The `grafana-sidecar-dashboard.json` template shows Radius Deployment Engine status, including runtime and server-side health:
```
Comment (youngbupark) [inline docs/content/operations/control-plane/observability/metrics/grafana.md]: Radius doesn't have sidecar.

File: docs/content/operations/control-plane/observability/metrics/grafana.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+weight: 5000
+description: "How to view Radius metrics in Grafana dashboards."
+---
+
+## Available dashboards
+
+{{< tabs "Core RP & Link RP" "Deployment Engine" "UCP" >}}
+
+{{% codetab %}}
+The `grafana-system-services-dashboard.json` template shows Radius Core RP and Link RP statuses, including runtime, server-side, and async operation health:
```
Comment (youngbupark) [inline docs/content/operations/control-plane/observability/metrics/grafana.md]: Core RP & Link RP is not the term user can understand easily.  This document describes two dashboard. the first one is `Radius Overview` which monitors the overall radius service. the second one is `Resource Provider` which monitors resource provider's status. 

* Radius Overview - https://radius-longhaul-wus3-awejb5drbjabh8h0.wus3.grafana.azure.com/d/Wh5JpyxVz/radius-overview?orgId=1&from=1677877199000&to=1677887999000
* Resource Provider - https://radius-longhaul-wus3-awejb5drbjabh8h0.wus3.grafana.azure.com/d/Wh5JpyxVz/radius-overview?orgId=1&from=1677877199000&to=1677887999000

**Comment by AaronCrawfis** [comment]
Looks like `radius` was added as a submodule. Could you remove that please?

File: docs/content/operations/control-plane/observability/metrics/prometheus.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+```bash
+docker run \
+    --net=host \
+    -v /tmp/prometheus.yml:/etc/prometheus/prometheus.yml \
+    prom/prometheus --config.file=/etc/prometheus/prometheus.yml --web.listen-address=:8080
+```
+
+Once Prometheus is running, you'll be able to visit its dashboard by visiting `http://localhost:8080`.
+
+## Setup Prometheus on Kubernetes
```
Comment (AaronCrawfis) [inline docs/content/operations/control-plane/observability/metrics/prometheus.md]: For Kubernetes installations does it automatically have the correct metrics port from the Radius Control Plane? Just making sure that the lack of configuration is intended, as the local installation instructions make you specify the ports

File: docs/content/operations/control-plane/observability/metrics/grafana/index.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+   - Name: `Radius`
+   - HTTP URL: `http://radius-prom-prometheus-server.radius-monitoring`
+   - Default: On
+
+1. Click `Save & Test` button to verify that the connection succeeded.
+
+### Import dashboards in Grafana
+
+1. Download the [Grafana dashboard templates](#TBD)
```
Comment (AaronCrawfis) [inline docs/content/operations/control-plane/observability/metrics/grafana/index.md]: Have these been published to blob storage yet?

File: docs/content/operations/control-plane/observability/metrics/metrics-overview.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+weight: 200
+description: "Learn about the metrics that are omitted by the Project Radius control plane"
+slug: "overview"
+---
+
+The Project Radius control plane emits metrics that can be used to monitor the health of the control plane and the resources that it manages.
+
+## Available metrics
+
+TODO
```
Comment (AaronCrawfis) [inline docs/content/operations/control-plane/observability/metrics/metrics-overview.md]: Could we add a table with the metrics we collect? Not sure how big of a table that would be.

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**